packages/suir-component-mapper/src/files/field-array.js

```diff
@@ -0,0 +1,269 @@
+          remove(value.length - 1);
+          dispatch({ type: 'redo' });
+        };
+
+        return (
+          <div>
+            <div className={classes.arrayHeader}>
+              {label && (
+                <FormField
+                  className={classes.noMargin}
+                  error={
+                    isError && {
+                      content: error,
+                      color: 'red',
+                      pointing: 'left'
+                    }
+                  }
+                  control={() => (
+                    <Header
+                      floated="left"
+                      className={clsx(classes.noMargin, {
+                        [classes.error]: isError
+                      })}
+                      size="large"
+                    >
+                      {label}
+                    </Header>
+                  )}
+                />
+              )}
+              <div className={classes.buttonGroup}>
+                <ButtonGroup>
+                  <Button className="ddorg__suir__mapper__field-array-undo" type="button" disabled={state.index === 0} onClick={undo}>
+                    <ButtonContent>
+                      <Icon name="undo" />
+                    </ButtonContent>
+                  </Button>
+                  <Button
+                    className="ddorg__suir__mapper__field-array-redo"
+                    type="button"
+                    disabled={state.index === state.history.length}
+                    onClick={redo}
+                  >
+                    <ButtonContent>
+                      <Icon name="redo" />
+                    </ButtonContent>
+                  </Button>
+                  <Button
+                    type="button"
+                    content={combinedButtonLabels.add}
+                    icon="add"
+                    color="blue"
+                    onClick={pushWrapper}
+                    disabled={value.length >= maxItems}
+                  />
+                </ButtonGroup>
+              </div>
+            </div>
+            {description && (
+              <Header className={classes.noMargin} sub>
+                {description}
+              </Header>
+            )}
+            <div className={classes.arrayItems}>
+              {value.length <= 0 ? (
+                <p className={classes.noItems}>{noItemsMessage}</p>
+              ) : (
+                map((name, index) => (
+                  <ArrayItem
+                    key={name}
+                    fields={formFields}
+                    name={name}
+                    fieldIndex={index}
+                    remove={removeWrapper}
+                    length={value.length}
+                    minItems={minItems}
+                    removeLabel={combinedButtonLabels.remove}
+                  />
+                ))
+              )}
+            </div>
+            {isError && (
+              <div className="ddorg__suir__mapper__field-array-error">
```

Comment:
some classname here

Reply:
I added this for tests so it can be easily found. I want to avoid using generic ids on generated content. But I can see this HTML element being replaced by some SUIR layout component very soon.